python/cudf/cudf/_lib/concat.pyx

```diff
@@ -30,6 +30,9 @@ def _column_concat(cols_to_concat, Column output_col):
             num_cols
         )
 
+    if hasattr(col, "null_count"):
```

Comment:
`col` is an intermediate looping variable as far as I can tell here. Should this be `output_col`?